bcipy/gui/bciui.py

```diff
@@ -0,0 +1,324 @@
+import os
```

Comment:
`gui.py` or `main.py` are possible filenames. This one works though. We should deprecate older files in this PR. If the filename changes, we'll also need to update the Makefile!